components/dashboard/src/workspaces/WorkspaceEntry.tsx

```diff
@@ -64,28 +74,19 @@ export const WorkspaceEntry: FunctionComponent<Props> = ({ info, shortVersion })
                         </div>
                     </a>
                 </Tooltip>
-            </ItemField>
+            </div>
             {!shortVersion && (
                 <>
-                    <ItemField className="w-4/12 flex flex-col my-auto">
-                        <div className="text-gray-500 dark:text-gray-400 overflow-ellipsis truncate">
-                            {workspace.metadata!.name}
-                        </div>
-                        <a href={normalizedContextUrl}>
```

Comment:
Is there another way for users to learn about the contextURL their workspace was started on? IMO we should at least have it discoverable somewhere, e.g. on hover or similar.

Reply:
I believe the context URL is missleading as it is only used in order to create the workspace initially, but doesn't reflect the state of the workspace as the user works in it. I think the git status is the relevant information.

Reply:
Generally true, but I was thinking about the special cases (snapshot etc.)

let's move on here anyway, we can solve this later.